plugins/experimental/stale_while_revalidate/stale_while_revalidate.c

```diff
@@ -81,46 +78,23 @@ typedef struct {
   TSIOBufferReader req_io_buf_reader, resp_io_buf_reader;
   TSVIO r_vio, w_vio;
   TSVConn vconn;
-  RequestInfo *req_info;
-  ResponseInfo *resp_info;
+  RequestInfo req_info_obj, *req_info;
+  TSHttpStatus rmt_resp_status;
+  time_t rmt_resp_date;
   time_t txn_start;
   config_t *plugin_config;
 } StateInfo;
 
-static ResponseInfo *
```

Comment:
doing a "manual" parse of the status-line is far simpler .. and very standard, only to find the response-status number.

ResponseInfo and all related parsing can be removed with that.